packages/openapi-generator/src/schema-util.ts

```diff
@@ -8,24 +8,54 @@ import {
   OpenApiNotSchema,
   OpenApiObjectSchema,
   OpenApiOneOfSchema,
+  OpenApiReferenceSchema,
   OpenApiSchema
 } from './openapi-types';
 
 /**
- * Collect all reference paths within a schema.
+ * Collect all reference schemas within a schema.
  * @param schema Parsed schema to retrieve all references for.
- * @returns Returns a list of reference paths within a schema.
+ * @returns Returns a list of unique reference schemas within a schema.
  */
-export function collectRefs(schema: OpenApiSchema | undefined): string[] {
+export function collectRefs(
+  schema: OpenApiSchema | undefined
+): OpenApiReferenceSchema[] {
+  return getUniqueRefs(collectAllRefs(schema));
+}
+
+/**
+ * Collect all reference schemas within a schema.
```

Comment:
The descriptions of these two method are the same, and the function names do not really tell the differences as they are both plural. 
Maybe add a keyword `unique` to the description and the name of the `collectRefs`?

Reply:
Added this to the docs